CONTRIBUTING.md

```diff
@@ -96,4 +104,19 @@ And more. The updated list of external resources can be found in [tools/fetch-ex
 
 Version numbers and commit messages are automatically added to the `CHANGELOG.md` file by the CI build flow after a pull request is merged. It should not be manually edited.
 
+## [balenistas only] Using Versionbot to manage your pull request
+
+Verisionbot is a helper utility to make managing of your pull request easier. Here's a list of commands supported by VersionBot:
+
+- `@balena-ci I self-certify!` - use [the self-certification flow](https://docs.google.com/drawings/d/1aoXCdi8bzinxaLM0BoKMJaiZjIJj5Az02Npw_L2iQkE/edit?usp=sharing) for this PR
+- `@balena-ci test` - trigger the tests for this PR
+- `@balena-ci retest` - trigger retests for the PR
+- `@balena-ci version` - generate the versioned branch for your PR
+- `@balena-ci rebase` - rebase the PR (Use this to update your branch)
+
+![](https://user-images.githubusercontent.com/22801822/114602349-22fd3d00-9cb4-11eb-905b-721488e64dc6.png)
+
+One can use these commands to manage, rebase or merge their own PR's as and when needed. We recommend using these commands to merge the PR if it's ready to be merge. One can issue the commands in the comments dialog box present at the bottom of the PR. 
```

Comment:
"ready to be merge" -> "ready to be merged" ?

Reply:
Roger that, will fix in the rest PR. 
I have to move some content from the README onto contributing. So that folks need to read only one document while going through the process of contributing.